tests/testdata/func_test_es_cluster.yaml

```diff
@@ -13,4 +13,4 @@ esconfig:
     scheme: "http"
     host: "${ES_SEEDS}:9200"
   indices:
-    visibility: temporal_visibility_v1_integration_test
+    visibility: temporal_visibility_v1_func_test
```

Comment:
Just use `test` instead of `integration_test` in all configs. It is implicit that these tests are "functional".